templates/index.md.tmpl

```diff
@@ -32,6 +32,58 @@ The changelog for this provider can be found here: <https://github.com/grafana/t
 
 {{ .SchemaMarkdown | trimspace }}
 
+### Managing Cloud Provider
+
+Before using the cloud provider, you need to create an access policy token on the Grafana Cloud Portal. This token is used to authenticate the provider to Grafana's Cloud Provider API.
```

Comment:
can we add a #### sub-header to separate

1. the instructions for getting the access token 
2. the instructions for getting the URL 
3. the instructions for configuring the provider

Reply:
added